plugin/plugin.go

```diff
@@ -48,9 +73,14 @@ func CreateSRK(tpm transport.TPMCloser) (*tpm2.AuthHandle, *tpm2.TPMTPublic, err
 // returns the identity and the corresponding recipient.
 // Note: It does not load the identity key into the TPM.
 func CreateIdentity(tpm transport.TPMCloser, pin []byte) (*Identity, *Recipient, error) {
-	srkHandle, srkPublic, err := CreateSRK(tpm)
+	srkHandle, srkPublic, err := getSharedSRK(tpm)
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed creating SRK: %v", err)
+		fmt.Fprintf(os.Stdout, "failed to acquire shared SRK, falling back to creating transient SRK: %v\n", err)
```

Comment:
Remove this or add it as output to `plugin.Log.Println`.

Reply:
Addressed in latest push